RIGImageGalleryDemo/View Controller/ViewController.swift

```diff
@@ -127,12 +127,20 @@ private extension ViewController {
 
         let items: [UIImage] = ["1", "2", "3", "4", "5", "6"].flatMap(UIImage.init(named:))
 
-        let rigItems = items.map { item in
-            RIGImageGalleryItem(image: item)
+        let rigItems: [RIGImageGalleryItem] = items.map { item in
+            var item = RIGImageGalleryItem(image: item)
+            item.isLoading = true
+            return item
         }
 
         let rigController = RIGImageGalleryViewController(images: rigItems)
 
+        DispatchQueue.main.asyncAfter(deadline: DispatchTime.now() + DispatchTimeInterval.seconds(2)) {
+            for index in 0..<rigController.images.count {
+                rigController.images[index].isLoading = false
```

Comment:
#? So to get the loading indicators working in practice, you'd just add this call once you're done loading, correct? If so, I'd just add that to the readme example.